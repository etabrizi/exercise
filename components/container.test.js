import * as React from 'react';
import Container from './Container';
import renderer from 'react-test-renderer';

const Props = {
    img: 'path-to-img-url',
    infoToggle: true
}

describe('Container component', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<Container {...Container} />)
            .toJSON();
        expect(tree).toMatchSnapshot()
    });
});

