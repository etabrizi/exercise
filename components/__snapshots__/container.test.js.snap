// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component renders correctly 1`] = `
<div
  className="jsx-3869787912"
  id="main-container"
/>
`;
